commands/meme.js

```diff
@@ -0,0 +1,51 @@
+const Augur = require("augurbot"),
+    u = require("../utils/utils");
+
+const Module = new Augur.Module()
+    .addCommand({
+        name: "meme",
+        category: "Meme",
+        description: "Creates a meme, put an image URL for you background and then put the text you want along the bottom. Or put the image source afterwards. Who am I to judge?",
+        permissions: (msg) => msg.channel.type === 'dm' || msg.channel.permissionsFor(msg.member).has(["ATTACH_FILES", "EMBED_LINKS"]),
+        process: (msg) => {
+            let { suffix } = u.parse(msg, true);
+            //general globals from bot this was imported from
+            const args = suffix.trim().split(/\n/);
```

Comment:
Let's avoid regex when possible. (I know, I didn't point it out earlier.)
```suggestion
            const args = suffix.trim().split("\n");
```